views/css/order.css

```diff
@@ -192,35 +195,18 @@
   content: "\e901";
 }
 
-.bnomics-rotate-btc {
-  -webkit-transform:rotate(15deg);
-  -moz-transform: rotate(15deg);
-  -ms-transform: rotate(15deg);
-  -o-transform: rotate(15deg);
-  transform: rotate(15deg);
-  margin-left: auto;
-  margin-right: auto;
-}
-
-.bnomics-rotate-bch {
-  -webkit-transform:rotate(-15deg);
-  -moz-transform: rotate(-15deg);
-  -ms-transform: rotate(-15deg);
-  -o-transform: rotate(-15deg);
-  transform: rotate(-15deg);
-  margin-left: auto;
-  margin-right: auto;
-}
-
 .bnomics-select-options {
   cursor:pointer;
-  width:50%;
+  width:100%;
+  display: block;
+  height: 4.2em;
```

Comment:
More themes should be tested to ensure the new layout works with different themes.

For instance [this theme](https://templatetoaster.com/template/fhoto-studio-online-animals-photography-prestashop-theme) includes the following CSS
```
.btn:hover {
    clear: none;
    height: auto;
}
```
This leads to the size of the button changing on hover. We can consider using min-height instead.

Reply:
I think we should remove height altogether. If the buttons are supposed to match the site's theme, they should match the inner paddings, and height as well.

Reply:
We can remove the height altogether, however this should not cause issues with the icons or vertical line on smaller sized buttons.

Reply:
I tested it with 3 themes after removing the height attribute, the button is adjusting itself with the height required.
The vertical line is just a border so it'll never overflow.

Reply:
Okay, lets remove the height attribute, `<tr>`, and the commented out border-radius/text-transform.

Reply:
Okay